docs/ReleaseNotes.md

```diff
@@ -1,6 +1,23 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.2
+Acuminator 2.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.2, the following enhancements have been implemented:
+ - The [PX1019](diagnostics/PX1019.md) diagnostic has been added to check for DAC fields of non-strings types with the `AutoNumberAttribute` attribute.
+ - The [PX1020](diagnostics/PX1020.md) diagnostic has been added to check if the length of a string DAC field with the `AutoNumberAttribute` attribute is sufficient.
+ - The [PX1022](diagnostics/PX1022.md) diagnostic has been added to check if a DAC or graph extension is declared as a public class.
+ - The context menu of Acuminator has been updated: Sort commands have been divided in the context menu, active sort indicator has been added, and menu control templates have been redesigned to integrate the Visual Studio theme.
```

Comment:
There were no sort commands at all before this version, so, I guess it should be reworded. 
It's better to write that sort commands were added, and you can also add that context menu layout was redesigned to better integrate with Visual Studio style

Reply:
I would also add a place where it changed - in a Code Map. Something like this:
The context menu of Acuminator Code Map has been updated